impl/maven-core/src/test/java/org/apache/maven/artifact/handler/ArtifactHandlerTest.java

```diff
@@ -77,8 +107,16 @@ void testAptConsistency() throws Exception {
                 ArtifactHandler handler =
                         container.lookup(ArtifactHandlerManager.class).getArtifactHandler(type);
                 assertEquals(handler.getExtension(), extension, type + " extension");
-                // Packaging/Directory is Maven1 remnant!!!
-                // assertEquals(handler.getPackaging(), packaging, type + " packaging");
```

Comment:
better some special workaround assert than no at all!

Agree upon test code allowed some flexibility like having **commented out code**. So lets use this liberty to improve the test by testing the production code.

This topic remains, but we have cleared up the doing and increased coverage.

If/as no ability to fix it yet, might consider take the increment and go for next increment.